omnigan/losses.py

```diff
@@ -466,7 +479,7 @@ def __call__(self, prediction, target):
 
 class ADVENTAdversarialLoss(nn.Module):
     """
-        TODO
+        The first and second arguement are tensor. The third arguement is a discriminator model.
```

Comment:
This is a class level docstring so it should explain what's the goal of the class itself, why does it exist, what does it do, in what context. The details (of arguments for instance) should be in method docstrings (like `__call__`'s)